README.md

```diff
@@ -17,8 +17,14 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v2-beta
+      with:
+        fetch-depth: 0
     - uses: simplabs/ember-asset-size-action@v1
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
 ```
+
+Note: as this action requires access to the "base" commit of a PR branch we need to fetch the whole repo by adding `fetch-depth: 0` to the `actions/checkout` configuration.
+
+If you would like to follow discussion on this issue you can follow: https://github.com/actions/checkout/issues/93
```

Comment:
duplicate "follow" here makes the sentence sound a little strange 😉

Reply:
fixed 👍